jaxrs-api/src/main/java/javax/ws/rs/JAXRS.java

```diff
@@ -0,0 +1,375 @@
+/*
+ * Copyright (c) 2018 Markus KARG. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+package javax.ws.rs;
+
+import java.util.concurrent.CompletionStage;
+
+import javax.ws.rs.core.Application;
+import javax.ws.rs.ext.RuntimeDelegate;
+
+/**
+ * Bootstrap class that is used to startup a JAX-RS application in Java SE
+ * environments.
+ * <p>
+ * The {@code JAXRS} class is available in a Jakarta EE container environment as
+ * well; however, support for the Java SE bootstrapping APIs is <em>not
+ * required</em> in container environments.
+ * </p>
+ * <p>
+ * In a Java SE environment an application is getting started by the following
+ * command using default configuration values (i. e. mounting application at
+ * {@code http://localhost:8080/} or a different port (there is no default port
+ * mandated by this specification):
+ * </p>
+ *
+ * <pre>
+ * JAXRS.start(new MyApplication(), JAXRS.Configuration.builder().build());
+ * </pre>
+ * <p>
+ * The following example shows how to override default values for protocol,
+ * host, port and root path:
+ * </p>
+ *
+ * <pre>
+ * JAXRS.Configuration config = JAXRS.Configuration.builder().protocol("HTTP").host("0.0.0.0").port(80).rootPath("api")
```

Comment:
We could add a few newlines here, so that each builder step is in a separate line. This makes it much easier to read.

Reply:
I will replace the complete description once the API is stabilized.